# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/miniupnpd/create-configdir.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- miniupnpd-2.3.11/Makefile.linux.orig
+++ miniupnpd-2.3.11/Makefile.linux
@@ -109,4 +109,5 @@
	$(STRIP) miniupnpd
	$(INSTALL) -D -m 0755 miniupnpd -t $(DESTDIR)$(SBINDIR)
+	mkdir -p $(DESTDIR)$(CONFIGDIR)
	$(INSTALL) -D -m 0755 $(NETFILTER_SCRIPTS) -t $(DESTDIR)$(CONFIGDIR)
	$(INSTALL) -D -m 0644 -b $(SRCDIR)/miniupnpd.conf -t $(DESTDIR)$(CONFIGDIR)

@@ -124,5 +125,6 @@
 else
	$(INSTALL) -D -m 0755 $(SRCDIR)/linux/miniupnpd.init.d.script $(DESTDIR)$(INITDIR)/miniupnpd
 endif
+	mkdir -p $(DESTDIR)$(MANDIR)/man8
 	$(INSTALL) -D -m 0644 $(SRCDIR)/miniupnpd.8 -t $(DESTDIR)$(MANDIR)/man8
